partiql/partiql-lang-rust
# Update Conformance Test Generator to latest partiql-tests

- Updates to latest partiql-tests
  - update partiql-tests submodule
  - update schema parsing to current schema
  - ignore eval tests for now

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
